open_lm/main.py

```diff
@@ -375,6 +379,7 @@ def main(args):
     args.wandb = "wandb" in args.report_to or "all" in args.report_to
     args.tensorboard = "tensorboard" in args.report_to or "all" in args.report_to
     args.checkpoint_path = os.path.join(log_base_path, "checkpoints")
+    args.failed_checkpoint_path = os.path.join(log_base_path, "checkpoints_failed")
```

Comment:
don't love adding more things to args but approving for now, in the future let's try to make new variables and pass them around